modules/comments/comments.php

```diff
@@ -334,8 +336,26 @@ public function comment_form_after() {
 				</h3>
 			<?php endif; ?>
 			<form id="commentform" class="comment-form">
-				<iframe title="<?php esc_attr_e( 'Comment Form', 'jetpack' ); ?>" src="<?php echo esc_url( $url ); ?>" style="width:100%; height: <?php echo $height; ?>px; border:0;" name="jetpack_remote_comment" class="jetpack_remote_comment" id="jetpack_remote_comment" sandbox="allow-same-origin allow-top-navigation allow-scripts allow-forms allow-popups"></iframe>
-				<?php if ( ! Jetpack_AMP_Support::is_amp_request() ) : ?>
+				<iframe
+					title="<?php esc_attr_e( 'Comment Form', 'jetpack' ); ?>"
+					src="<?php echo esc_url( $url ); ?>"
+					<?php if ( $is_amp ) : ?>
+						resizable
+						layout="fixed-height"
+						height="<?php echo esc_attr( $height ); ?>"
+					<?php else : ?>
+						name="jetpack_remote_comment"
+						style="width:100%; height: <?php echo $height; ?>px; border:0;"
```

Comment:
Should we take that opportunity to add some escaping?

```suggestion
						style="width:100%; height: <?php echo esc_attr( $height ); ?>px; border:0;"
```